### PR TITLE
95 architecture decisions

### PR DESCRIPTION
**Pull request by rqding**
See seedcase-project/design#31

### CODE REVIEW

**Comment by lwjohnst86** [comment]
:raised_hands: File names need to be `index.md` though. To make it easier for reviewing, please run `bash src/pr/reformat-markdown.sh` to reformat the files into canonical markdown mode :star_struck:

File: design/decisions/why-REST/why-rest.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## REST API for client-server communication
+### Decision
+We will use the REST (Representational State Transfer) architectural style for
+implementing the communication protocol between our client and server
+components, instead of SOAP (Simple Object Access Protocol).
```
[the remark this answers is not in the document]
Comment (rqding) [reply design/decisions/why-REST/why-rest.md:5]: Sure, i will add some more for the GraphQL

**Comment by lwjohnst86** [comment]
@rqding any progress on these items?

**Comment by rqding** [comment]
Sorry Luke forget to push my local changes

**Pull request merged**